src/main/java/io/cryostat/rules/ArchivedRecordingInfo.java

```diff
@@ -67,6 +68,22 @@ public ArchivedRecordingInfo(
         this.downloadUrl = downloadUrl;
         this.reportUrl = reportUrl;
         this.metadata = metadata;
+        this.size = 0;
```

Comment:
This constructor is also used in a unit test, so similar comment as the previous.

It's also used in `RecordingsPostHandler` which is the endpoint for re-uploading JFR files into Cryostat's archives (related: https://github.com/cryostatio/cryostat/pull/1047) and `PutArchivedRecordingMetadataMutator`, which is the GraphQL mutator for updating metadata/labels on archived recordings.

When a file is uploaded to Cryostat's archives, we can easily check the size of that file and use a real value here rather than `0`.

In the case of updating metadata related to an archived recording, the mutator actually already has an `ArchivedRecordingInfo` context object to begin with. These are immutable and so the mutator returns a new object with copied values and an updated/replaced `Metadata`. The size field can also just be copied from the context object.

Reply:
done!
In the tests, I just passed 0 value to file size.